lib/tasks/temporary/create_profile_fields.rake

```diff
@@ -1,7 +1,8 @@
 namespace :temporary do
-  desc "Create profile fields based on DEV profile" 
-  task create_profile_fields: :environment do
-    csv = Rails.root.join("lib/data/dev_profile_fields.csv")
-    ProfileFields::ImportFromCsv.call(csv)
+  desc "Migrate existing profile data"
+  task migrate_profiles: :environment do
+    User.find_each do |user|
+      Profile.create(user: user, data: Profiles::ExtractData.call(user))
```

Comment:
Should this be in a data update script so it can run on all environments or are we not quite there yet?

Reply:
A data update script automatically gets scheduled after deployment, I'd much rather have someone start this by hand at a good time or run `Profile.create(user: user, data: Profiles::ExtractData.call(user))` for smaller batches as they see fit.

Reply:
How do you plan on making sure this is run on all instances of Forem then? We have Lee, Forem.dev and who knows what other communities out there that will need this run.

Reply:
@mstruve I figured we can delay this until the feature is completely done but you just changed my mind on this 👍